### PR TITLE
[docs] PR for Restructure doc site wrt Ecosystem Integration (Doc8860)

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/latest/integrations/_index.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+section: INTEGRATIONS
 aliases:
 menu:
   latest:
-    identifier: ecosystem-integrations
-    parent: develop
-    weight: 570
+    identifier: integrations
+    weight: 640
 ---
```
Comment (ddhodge) [inline docs/content/latest/integrations/_index.html:14]: Match tile order with menu order (or vice versa)

File: docs/content/latest/integrations/_index.html [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-title: Ecosystem integrations using YugabyteDB
-headerTitle: Ecosystem integrations
-linkTitle: Ecosystem integrations
+title: Integrations
+linkTitle: Integrations
 description: Integrate YugabyteDB with Apache Kafka, Apache Spark, JanusGraph, KairosDB, Presto Metabase, WSO2 IS, Hasura
```
Comment (ddhodge) [inline docs/content/latest/integrations/_index.html:4]: Some missing from description
headContent - "a few of the ecosystem of"- well we're dropping 'ecosystem ' anyways, so cut or rewrite?

**Review by ddhodge — COMMENTED** [review]
Minor comments, some broken links

File: docs/content/latest/integrations/graphql-hasura.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ aliases:
 menu:
   latest:
     identifier: graphql-hasura
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/graphql-hasura.md:8]: I'm not sure about aliases. I've heard that we don't want them, but should have in some cases.
Maybe when I close this PR, Alex can look into this.

File: docs/content/latest/integrations/hasura.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ description: Hasura GraphQL engine
 menu:
   latest:
     identifier: hasura
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/hasura.md:7]: I think this file is obsolete. It's now hidden. I'll check with dev to see if any of its content is worth keeping, in which case I'll merge it with graphql-hasura.md

File: docs/content/latest/develop/_index.md [hunk 5]
```diff
@@ -30,19 +30,6 @@ menu:
     </a>
   </div>
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/develop/_index.md:32]: Fixed. Thanks for catching this

File: docs/content/latest/integrations/_index.html [hunk 6]
```diff
@@ -32,7 +31,7 @@
         <div class="title">Apache Spark</div>
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/_index.html:31]: Fixed

File: docs/content/latest/integrations/_index.html (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/_index.html:4]: Fixed

File: docs/content/latest/integrations/_index.html (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/_index.html:14]: Fixed

**Comment by lizayugabyte** [comment]
@polarweasel Would you like to take a look?

File: docs/content/latest/integrations/graphql-hasura.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/latest/integrations/graphql-hasura.md:8]: We need to alias anything that is linked FROM anywhere else. I expect that quite a few of these pages are linked from blogs, so we don't want those to break. Aliases can be server-side if there are just a few, or per-page if the relative arrangement of every page has changed.

File: docs/content/latest/integrations/hasura.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/latest/integrations/hasura.md:7]: yeah, we were going to merge the two... definitely check on what's worth keeping here.

Also, "hidden" pages still show up in the sitemap, so they still get indexed: have a look at https://deploy-preview-9105--infallible-bardeen-164bc9.netlify.app/sitemap.xml and search for "hasura" to see what I mean.

[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/hasura.md:7]: Sure I can find this file if i add it the the URL. But I don't think it's linked from anywhere, so it's not very likely for a user to open it. I want to create a new ticket for the hasura files (merge the two or delete the old one as obsolete)

**Comment by lizayugabyte** [comment]
Created https://github.com/yugabyte/yugabyte-db/issues/9130 for hasura.md

File: docs/content/latest/integrations/metabase.md [hunk 7]
```diff
@@ -46,7 +48,7 @@ orders.sql  products.sql  reviews.sql  users.sql
 
 ### Connect to YugabyteDB using ysqlsh
 
-You can do this as shown below.
+You can do this by executing the following:
```
Comment (polarweasel) [inline docs/content/latest/integrations/metabase.md:51]:
```suggestion
Run the following command to connect to YugabyteDB using the YSQL shell:
```

File: docs/content/latest/integrations/metabase.md [hunk 8]
```diff
@@ -105,6 +107,7 @@ Detailed steps on how to use Metabase are available in the [Metabase documentati
 - Filter data using WHERE clauses
 - Join data between tables
 - Perform data aggregation using GROUP BY
-- Use built-in functions such as SUM, MIN, MAX, etc.
+- Use built-in functions such as SUM, MIN, MAX, and so on
+
+Click **Ask a Question > Custom Query**. Choose the database you just set up, and enter the SQL queries noted in the [Retail Analytics](../../develop/realworld-apps/retail-analytics/) section.
```
Comment (polarweasel) [inline docs/content/latest/integrations/metabase.md:112]: I wonder if the Retail Analytics section could be combined with this one?

File: docs/content/latest/integrations/presto.md [hunk 9]
```diff
@@ -63,7 +63,7 @@ node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
 node.data-dir=/Users/<username>/presto-server-309/data
 ```
 
-Press Ctrl-D after you've pasted the file contents.
+Press Ctrl-D after you have pasted the file contents.
```
Comment (polarweasel) [inline docs/content/latest/integrations/presto.md:66]: Contractions are usually good...

```suggestion
Press Ctrl-D after you've pasted the file contents.
```

File: docs/content/latest/integrations/presto.md [hunk 10]
```diff
@@ -82,7 +82,7 @@ $ cat > etc/jvm.config
 -XX:+ExitOnOutOfMemoryError
 ```
 
-Press Ctrl-D after you've pasted the file contents.
+Press Ctrl-D after you have pasted the file contents.
```
Comment (polarweasel) [inline docs/content/latest/integrations/presto.md:85]: Same as previous :) 
```suggestion
Press Ctrl-D after you've pasted the file contents.
```

File: docs/content/latest/integrations/presto.md [hunk 11]
```diff
@@ -100,7 +100,7 @@ discovery-server.enabled=true
 discovery.uri=http://localhost:8080
 ```
 
-Press Ctrl-D after you've pasted the file contents.
+Press Ctrl-D after you have pasted the file contents.
```
Comment (polarweasel) [inline docs/content/latest/integrations/presto.md:103]:
```suggestion
Press Ctrl-D after you've pasted the file contents.
```

File: docs/content/latest/integrations/presto.md [hunk 12]
```diff
@@ -112,11 +112,11 @@ $ cat > etc/log.properties
 io.prestosql=INFO
 ```
 
-Press Ctrl-D after you've pasted the file contents.
+Press Ctrl-D after you have pasted the file contents.
```
Comment (polarweasel) [inline docs/content/latest/integrations/presto.md:115]:
```suggestion
Press Ctrl-D after you've pasted the file contents.
```

File: docs/content/latest/integrations/presto.md [hunk 13]
```diff
@@ -128,7 +128,7 @@ connector.name=cassandra
 cassandra.contact-points=127.0.0.1
 ```
 
-Press Ctrl-D after you've pasted the file contents.
+Press Ctrl-D after you have pasted the file contents.
```
Comment (polarweasel) [inline docs/content/latest/integrations/presto.md:131]:
```suggestion
Press Ctrl-D after you've pasted the file contents.
```

File: docs/content/latest/integrations/prisma.md [hunk 14]
```diff
@@ -5,15 +5,17 @@ description: Prisma
...
-    parent: graphql
-    weight: 583
+    parent: integrations
+    weight: 579
 isTocNested: true
 showAsideToc: true
 ---
 
-Explore how you can use Prisma, and its GraphQL support, to interact with YugabyteDB. You will quickly build a GraphQL server and then use the Prisma client to write data to and run queries on a YugabyteDB database. Also, you'll get a taste of Prisma's ORM functionality.
+Explore how you can use Prisma and its GraphQL support to interact with YugabyteDB. You can build a GraphQL server and then use the Prisma client to write data to and run queries on a YugabyteDB database. Also, you can explore the Prisma's ORM functionality.
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:14]: Taking out the "quickly" changes the flavour of this one a bit more than we might want. I think we probably want to convey that it's quick and easy to get this set up; "you can build a GraphQL server" does not make me think it's going to be easy, from what I've experienced of GraphQL. 😆

File: docs/content/latest/integrations/prisma.md [hunk 15]
```diff
@@ -25,11 +27,11 @@ If YugabyteDB is installed, run the following `yb-ctl create` command to start a
 ./bin/yb-ctl create --tserver_flags=ysql_pg_conf="default_transaction_isolation=serializable"
 ```
 
-If you are new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](/latest/quick-start/). After installing YugabyteDB, make sure to follow the step mentioned above.
+If you are new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](../../quick-start/). After installing YugabyteDB, perform the preceding step.
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:30]: Does this make more sense to have at the top of the section, instead?

File: docs/content/latest/integrations/prisma.md [hunk 16]
```diff
@@ -84,7 +86,7 @@ You should now have a `prismagraphql/prisma` container running. You can check th
 
 ## 3. Set up a sample schema
 
-Open `datamodel.prisma` and replace the contents with:
+Open `datamodel.prisma` and replace the contents with the following:
 
 ```
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:91]:
````suggestion
```graphql
````

File: docs/content/latest/integrations/prisma.md [hunk 17]
```diff
@@ -306,27 +308,27 @@ node index.js
 
 ## Clean up
 
-Now that you're done with this exploration, you can clean up the pieces for your next adventure.
+Now that you have completed with this exploration, you can clean up the pieces as follows:
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:311]:
```suggestion
Now that you've completed this exploration, clean up the pieces as follows:
```

File: docs/content/latest/integrations/prisma.md [hunk 18]
```diff
@@ -306,27 +308,27 @@ node index.js
...
 ```
 
-To completely remove all YugabyteDB data/cluster-state you can instead run:
+To completely remove all YugabyteDB data/cluster-state, you can instead run the following command:
 
 ```sh
 ./bin/yb-ctl destroy
 2. Stop The Prisma container
 docker stop <container-id>
 ```
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:325]: This ain't right...

File: docs/content/latest/integrations/prisma.md [hunk 19]
```diff
@@ -306,27 +308,27 @@ node index.js
...
 
-To completely remove all Prisma data/tate you can additionally run:
+To completely remove all Prisma data/state, you can additionally run the following command:
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:327]:
```suggestion
To completely remove all Prisma data and state, you can additionally run the following command:
```

File: docs/content/latest/integrations/prisma.md [hunk 20]
```diff
@@ -306,27 +308,27 @@ node index.js
...
 
 ```sh
 docker rm <container-id>
 ```
 
-Note: you can list running containers by running `docker ps`.
+To list running containers, execute `docker ps`.
```
Comment (polarweasel) [inline docs/content/latest/integrations/prisma.md:333]:
```suggestion
To list running containers, run `docker ps`.
```

File: docs/content/latest/integrations/wso2.md [hunk 21]
```diff
@@ -63,7 +63,7 @@ You update the database configuration as follows:
   ```
 
 
-- Set the following properties to `true` to configure cross-origin resource sharing (CORS) for running the sample application from [Quick Start](https://docs.google.com/document/d/1WJgV4Iu_kY06FDA1BiH8tx7IN_tTgyWz-Z2peoLfIWQ/edit#heading=h.cjh8ej37ta3t):
+- Set the following properties to `true` to configure cross-origin resource sharing (CORS) for running the sample application from [Quick Start](https://is.docs.wso2.com/en/latest/get-started/quick-start-guide/):
 
   ```
```
Comment (polarweasel) [inline docs/content/latest/integrations/wso2.md:68]:
````suggestion
  ```properties
````

**Review by polarweasel — CHANGES_REQUESTED** [review]
Comments and suggestions in many files...

File: docs/content/latest/integrations/graphql-hasura.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/graphql-hasura.md:8]: Okay, the decision was to automate aliases on Integrations

File: docs/content/latest/integrations/hasura.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/hasura.md:7]: We are not talking about forever

File: docs/content/latest/_index.html [hunk 22]
```diff
@@ -123,50 +115,46 @@
...
       <div class="head">
-        <img class="icon" src="/images/section_icons/index/deploy.png" aria-hidden="true" />
-        <div class="articles">6 articles</div>
-        <div class="title">Yugabyte Cloud</div>
+        <img class="icon" src="/images/section_icons/develop/ecosystem-integrations.png" aria-hidden="true" />
+        <div class="title">Integrations</div>
       </div>
       <div class="body">
-          Get started with a fully-managed YugabyteDB-as-a-Service on public cloud platforms.
+          Integrate YugabyteDB with Apache Kafka, Apache Spark, JanusGraph, KairosDB, Presto, Metabase, WSO2 IS, Hasura, Spring Framework, Prisma
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/_index.html:157]: I'd rather see a clear separation between tech docs and marketing.
All the marketing should be happening on https://yugabyte.zoom.us/ .
https://docs.yugabyte.com should be technical.

File: docs/content/latest/_index.html [hunk 23]
```diff
@@ -255,7 +240,7 @@
         <div class="title">Third party tools</div>
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/_index.html:240]: Killing them all is out of scope for this ticket.
I removed the ones that I noticed in the files I worked on for 8860

File: docs/content/latest/_index.html [hunk 24]
```diff
@@ -255,7 +240,7 @@
...
       </div>
       <div class="body">
-        GUI tools for developing and managing YugabyteDB databases. 
+        UI tools for developing and managing YugabyteDB databases
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/_index.html:243]: I don't like "GUI" in general (reminds me of the late 80s). But I'll reverse this one.

**Review by ddhodge — APPROVED** [review]

File: docs/content/latest/_index.html [hunk 25]
```diff
@@ -123,50 +115,46 @@
...
-    <a class="section-link icon-offset" href="yugabyte-platform/overview">
+    <a class="section-link icon-offset" href="yugabyte-platform/">
       <div class="head">
         <img class="icon" src="/images/section_icons/index/deploy.png" aria-hidden="true" />
-        <div class="articles">3 chapters</div>
         <div class="title">Yugabyte Platform</div>
       </div>
       <div class="body">
-          Deploy multi-node clusters on any public or private cloud. Includes manual, automated and orchestrated (using Kubernetes &amp; Docker Swarm) deployments.
+          Deploy multi-node clusters on any public or private cloud. Includes manual, automated and orchestrated (using Kubernetes &amp; Docker Swarm) deployments
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/_index.html:133]: Added comma

File: docs/content/latest/integrations/_index.html [hunk 26]
```diff
@@ -56,7 +56,7 @@
         <div class="title">KairosDB</div>
       </div>
       <div class="body">
-        Fast time series metrics API that uses Cassandra-compatible YugabyteDB as it's underlying distributed database.
+        Fast series metrics API that uses Cassandra-compatible YugabyteDB as its underlying distributed database.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/_index.html:58]: Fixed

File: docs/content/latest/integrations/prisma.md (hunk 17 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:311]: Changed to "Now that you have completed this exploration"

File: docs/content/latest/integrations/prisma.md (hunk 19 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:327]: Fixed

File: docs/content/latest/integrations/prisma.md (hunk 20 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:333]: Don't like two runs in s row. "execute" is better here.

File: docs/content/latest/integrations/wso2.md (hunk 21 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/wso2.md:68]: Set the code block to "properties"

File: docs/content/latest/integrations/prisma.md (hunk 18 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:325]: Good catch, it looks like this file was messed up a bit.
Fixed.

**Comment by lizayugabyte** [comment]
@polarweasel @ddhodge I'm merging this PR. We'll deal with aliases and any errors/improvements via new tickets.

**Comment by lizayugabyte** [comment]
Created https://github.com/yugabyte/yugabyte-db/issues/9140 to add aliases

File: docs/content/latest/integrations/_index.html [hunk 27]
```diff
@@ -109,4 +109,28 @@
...
   </div>
 
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="prisma/">
+      <div class="head">
+      <img class="icon" src="/images/develop/graphql/prisma/prisma.png" aria-hidden="true" />
+      <div class="title">Prisma</div>
+    </div>
+    <div class="body">
+      Use the Prisma client and the Prisma service (running in a Docker container) to add data and run queries on a YugabyteDB database. Also, take a quick look at the Prisma ORM functionality.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/_index.html:118]: Changed in DocMinorFixesForPR9105 PR

File: docs/content/latest/integrations/apache-spark/scala.md [hunk 28]
```diff
@@ -53,11 +51,11 @@ libraryDependencies += "com.yugabyte.spark" %% "spark-cassandra-connector" % "2.
 
 This tutorial assumes that you have:
 
-- installed YugabyteDB, created a universe, and are able to interact with it using the YCQL shell (`ycqlsh`). If not, please follow these steps in the [quick start guide](../../../../api/ycql/quick-start/).
+- installed YugabyteDB, created a universe, and are able to interact with it using the YCQL shell (`ycqlsh`). If not, please follow these steps in the [Explore YCQL](../../../api/ycql/quick-start/).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/apache-spark/scala.md:54]: Changed

File: docs/content/latest/integrations/apache-spark/scala.md [hunk 29]
```diff
@@ -53,11 +51,11 @@ libraryDependencies += "com.yugabyte.spark" %% "spark-cassandra-connector" % "2.
...
 
-- installed Scala version 2.12 or later and sbt 1.3.8 or later/
+- installed Scala version 2.12 or later and sbt 1.3.8 or later
 
-- installed the [`sbt-assembly`](https://github.com/sbt/sbt-assembly) plugin in your sbt project as shown below.
+- installed the [`sbt-assembly`](https://github.com/sbt/sbt-assembly) plugin in your sbt project , as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/apache-spark/scala.md:58]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/janusgraph.md [hunk 30]
```diff
@@ -16,7 +16,7 @@ In this tutorial, you are first going to setup [JanusGraph](https://janusgraph.o
 
 ## 1. Start local cluster
 
-Start a cluster on your [local machine](../../../quick-start/install/). Check that you are able to connect to YugabyteDB using `ycqlsh` by doing the following.
+Start a cluster on your [local computer](../../quick-start/install/). Check that you are able to connect to YugabyteDB using `ycqlsh` by executing the following:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/janusgraph.md:19]: This is a restructuring ticket.
Improvement of legacy content (not just here, but everywhere) should be done in new tickets (P4).

Split the code block.

And I'm not a fan of "machine" (wasn't allowed in OSG).

File: docs/content/latest/integrations/kairosdb.md [hunk 31]
```diff
@@ -28,13 +28,13 @@ $ tar xvfz kairosdb-1.2.0-0.3beta.tar.gz
 $ cd kairosdb/
 ```
 
-You can follow the [Getting started](http://kairosdb.github.io/docs/build/html/GettingStarted.html) to see how to configure KairosDB in general. For the purpose of integrating with the local YugabyteDB cluster running at `localhost:9042`, simply open `conf/kairosdb.properties` and comment out the default in-memory datastore as below.
+You can follow the [Getting started](http://kairosdb.github.io/docs/build/html/GettingStarted.html) to see how to configure KairosDB in general. For the purpose of integrating with the local YugabyteDB cluster running at `localhost:9042`, simply open `conf/kairosdb.properties` and comment out the default in-memory datastore, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/kairosdb.md:31]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/kairosdb.md [hunk 32]
```diff
@@ -73,7 +73,7 @@ Use HELP for help.
 ycqlsh>
 ```
 
-- Run a cql command to verify it is working.
+- Run a cql command to verify it is working:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/kairosdb.md:76]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/kairosdb.md [hunk 33]
```diff
@@ -60,7 +60,7 @@ You should see the following lines if KairosDB starts up successfully.
 
 Run `ycqlsh` to connect and use YugabyteDB's YCQL API.
 
-Assuming you are using the macOS or Linux binary
+Assuming you are using the macOS or Linux binary, execute the following:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/kairosdb.md:63]: It's fine as is.

File: docs/content/latest/integrations/metabase.md [hunk 34]
```diff
@@ -6,17 +6,19 @@ aliases:
...
+    parent: integrations
     weight: 576
 isTocNested: true
 showAsideToc: true
 ---
 
-[Metabase](https://www.metabase.com/) is an extremly easy-to-use Business Intelligence (BI) tool. It bills itself as `the easy, open source way for everyone in your company to ask questions and learn from data`. This page shows how Metabase can be setup to integrate with YugabyteDB's PostgreSQL compatible API.
+[Metabase](https://www.metabase.com/) is a Business Intelligence (BI) tool. 
+
+This document shows how Metabase can be set up to integrate with YugabyteDB's PostgreSQL-compatible API.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/metabase.md:17]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/metabase.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/metabase.md:51]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/metabase.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/metabase.md:112]: Please feel free to create an improvement ticket for that

File: docs/content/latest/integrations/presto.md (hunk 9 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/presto.md:66]: I respectfully disagree

File: docs/content/latest/integrations/presto.md (hunk 10 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/presto.md:85]: Same comment

File: docs/content/latest/integrations/prisma.md (hunk 14 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:14]: quickly, simply, easily, etc is marketing. When I see these things, I usually delete them from tech docs.

File: docs/content/latest/integrations/prisma.md (hunk 15 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:30]: Changed in DocMinorFixesForPR9105 branch

File: docs/content/latest/integrations/prisma.md (hunk 16 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/prisma.md:91]: graphql didn't stick in my editor

**Pull request merged**